benchmarks/run.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -xe
+
+WORKSPACE_ROOT="$( cd "$(dirname "$0")/.." ; pwd -P )"
+cd "$WORKSPACE_ROOT"
+
+# Benchmark the main branch
+tmpdir=$(mktemp -d)
+git worktree add "${tmpdir}" origin/main
```

Comment:
Can we change this to `git worktree add "${tmpdir}" main`? In CircleCI it should work fine, but in my local machine `origin/main` will usually point to an old version of the repo (I don't really update my fork ever). 

We can then mention somewhere in the README.md that the size check is against the local main branch.

```suggestion
git worktree add "${tmpdir}" main
```